run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+consts=""
+
+if [ -n "$1" ]; then
+  consts=-$1;
+fi;
+
+source constants$consts.sh
+
+docker run --name=$IMAGE -d -p $PORT:$EXPOSE_PORT -e PORT=$EXPOSE_PORT -e HOSTNAME=$HOSTNAME $DOCKER_USER$IMAGE:$TAG
```

Comment:
Could you please add a `--rm` or another option to allow re-running this script ? 
It would also be better to mount the project files as volumes so we don't need to rebuild after changing code.